web/book/src/queries/variables.md

```diff
@@ -34,7 +60,12 @@ let grouping = s"""
 from grouping
 ```
 
-## `into`
+```admonish info
+**Why introduce `into`?** It seems that `let = ...` completely covers the use case without introducing a new language feature.
+Does `into` work with scalars and functions?
+```
```

Comment:
Because it fits nicely with the concept of a pipeline:

```elm
from x
derive
filter
into y

from y
filter
join y
```

---

Yes, `into` works with everything that `let` does.